libc/include/llvm-libc-types/float16.h

```diff
@@ -0,0 +1,20 @@
+//===-- Definition of float16 type ----------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_TYPES_FLOAT16_H
+#define LLVM_LIBC_TYPES_FLOAT16_H
+
+#include "src/__support/macros/properties/compiler.h"
```

Comment:
I just realized, this is an internal header that we can't include in a public one, right? It isn't copied to the build directory.

Reply:
yes, you cannot include anything in the `src` folder from here.